src/read_load_time_mpi_constants_win64.jl

```diff
@@ -0,0 +1,279 @@
+# This file defines the load-time MPI constants in MicrosoftMPI. They
```

Comment:
Why is this file called `win64`? it's specific to Microsoft MPI (theoretically Intel MPI also works on Windows).

Reply:
I don't recall. I used the architecture name `win64` I found elsewhere. Maybe Yggdrasil? Or Github? `msmpi` might be a better name.

Reply:
Renamed.